addons/website_sale/static/src/js/website_sale.js

```diff
@@ -241,6 +246,10 @@ publicWidget.registry.WebsiteSale = publicWidget.Widget.extend(VariantMixin, car
         this._changeCountry = _.debounce(this._changeCountry.bind(this), 500);
 
         this.isWebsite = true;
+        this.startX = 0;
+        this.isDown = false;
+        this.scrollLeft = 0;
+        this.moved = false;
```

Comment:
Make it explicit these are related to the filmstrip drag mechanic
```suggestion
        // Filmstrip drag&drop
        this.filmStripStartX = 0;
        this.filmStripIsDown = false;
        this.filmStripScrollLeft = 0;
        this.filmStripMoved = false;
```
I agree those are long variable names but we are already working with a gigantic file handling a lot of different pages. It makes it way easier to identify and separate part of the code.